frontend/combination_order.py

```diff
@@ -0,0 +1,259 @@
+import pygame
+from frontend.image import Image
+from frontend.button import Button
+from frontend.loading import LoadingScreen
+import os
+from frontend.orders import Order
+
+# Set up the assets directory
+ASSETS_DIRECTORY = os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "assets", "frontend", "orders"))
+
+class CombinationOrder(Order):
+    COMPLETE, PENDING, DISCARDED = 0, 1, 2
```

Comment:
i would add a class docstring explaining what a CombinationOrder is an some examples (soup, sushi, etc.)